packages/babel-parser/test/fixtures/estree/module-string-names/mixed/options.json

```diff
@@ -1,3 +0,0 @@
-{
```

Comment:
The test options is moved to upper level `module-string-names/options.json` so it can be shared with the other test case.